ci/nightly/pipeline.template.yml

```diff
@@ -101,6 +101,28 @@ steps:
               - common-ancestor
               - --max-concurrency
               - "256"
+    - id: connection-initiation-benchmark
```

Comment:
In keeping with the style of the rest of the pipeline, I suggest you create a `scalability-benchmark` group and then the 2 (possibly more in the future) CI steps under it.

Reply:
Oh wait, it is already in the "benchmarking" group. So I guess no opportunity for more nesting. Please disregard the comment above.

Reply:
I indeed plan to create three separate jobs:
* scalability with selects
* scalability with connection
* scalability with DDL